docs/acme-divergences.md

```diff
@@ -46,6 +46,8 @@ Boulder does not implement the `meta` field returned by the `directory` endpoint
 
 Boulder does not implement the `terms-of-service-agreed` or `orders` fields in the registration object (nor the endpoints the latter links to).
 
+Boulder does not return the `status` field.
```

Comment:
I think this particular divergence is closer related to Section 6.3. "Account Creation". In Section 6.1.2 I don't see the status field included in the example response for a newly created account. Section 6.3 details how you get back an account object by POSTing, but it doesn't explicitly say which fields should be expected. It _seems_ reasonable that you'd expect to see the `status` field so I agree with the overall divergence if not the placement.

What do you think?